Library/Homebrew/cmd/info.rb

```diff
@@ -123,8 +123,12 @@ def info_formula(f)
     puts f.desc if f.desc
     puts Formatter.url(f.homepage) if f.homepage
 
-    conflicts = f.conflicts.map(&:name).sort!
-    puts "Conflicts with: #{conflicts*", "}" unless conflicts.empty?
+    conflicts = f.conflicts.map do |c|
+      c.name +
+        (c.reason ? " (because #{c.reason})" : "")
+    end.sort!
+    msg="Conflicts with: "
+    puts msg+conflicts*(",\n"+" "*msg.length) unless conflicts.empty?
```

Comment:
I've changed this to have a pretty simple solution that has the best of both, I think, and will work better on 80 character terminals. Thanks for caring, though ❤️